tests/harvest_api.py

```diff
@@ -102,230 +103,389 @@ def get_account_id():
 #     data = {"email":email,"first_name":"FirstName","last_name":"LastName","is_project_manager":True}
```

Comment:
Any reason for keeping the commented code?

Reply:
We do need that function for future purposes, Added a note in the code comment.